CLI examples cloudfront, cognito-idp, ec2, ecr-public, ecs, inspector… (#8960)
aws/aws-cli: awscli/examples/ecs/ecs-update-cluster.rst

**Example 1: Update ECS cluster enabling containerInsights**

> The following ``update-cluster`` example shows how we can update containerInsights value to enabled to the already created cluster. By default, it is disabled. ::

Reviewer: ```suggestion
The following ``update-cluster`` example sets the containerInsights value to enabled on the already created cluster. By default, it is disabled. ::
```
